### PR TITLE
[MODORDERS-989] - Implement batch endpoint to move multiple pieces to Expected status

### PR DESCRIPTION
**Pull request by Abdulkhakimov**
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?
- Did you modify code to call some additional endpoints?
  - [ ] If so, do you check that necessary module permission added in ModuleDescriptor-template.yaml file?

Ideally, all the PRs involved in breaking changes would be merged on the same day
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems,
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/helper/ExpectHelper.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.mapping;
+import static java.util.stream.Collectors.toList;
+
+public class ExpectHelper extends CheckinReceivePiecesHelper<ExpectPiece> {
```
Comment (SerhiiNosko) [inline src/main/java/org/folio/helper/ExpectHelper.java:27]: Can we rename to PieceService without extend operation with method moveToExpected(ExpectCollection expectCollection)

File: src/main/java/org/folio/helper/ExpectHelper.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
```
Comment (SerhiiNosko) [inline src/main/java/org/folio/helper/ExpectHelper.java:28]: please add separate logger for ExpectHelper

File: src/main/java/org/folio/helper/ExpectHelper.java (hunk 1 above)
Comment (SerhiiNosko) [reply src/main/java/org/folio/helper/ExpectHelper.java:27]: there is method for acq unit check in base class, also for future we potentially can be needed additional checks like to u[date statuses etc, so we decided to remain extends from CheckinReceivePiecesHelper

File: src/main/java/org/folio/helper/ExpectHelper.java (hunk 2 above)
Comment (Abdulkhakimov) [reply src/main/java/org/folio/helper/ExpectHelper.java]: Added, Thank you

**Review by SerhiiNosko — APPROVED** [review]

**Review by azizbekxm — APPROVED** [review]

**Pull request merged**